src/main/java/seedu/duke/Duke.java

```diff
@@ -1,6 +1,6 @@
 package seedu.duke;
 
-import java.util.Scanner;
+import java.util.*;
```

Comment:
Please follow coding standard
import java.util.Scanner instead of import java.util.*